tool/run/image/uefi

```diff
@@ -19,7 +19,7 @@ proc run_image { {unused ""} } {
 	if {[image_uefi_size] > 0} {
 		set disk_size  [image_uefi_size]
 	} else {
-		set disk_size [expr $run_size + 1]
+		set disk_size [expr $run_size + 20]
```

Comment:
What's your rationale to add 20M to `disk_size` here? IMO it's better to use the `--image--uefi--size` parameter for custom configurations.

Reply:
Initially I had the problem that the size of the generated image was too small for some cases. This is merely a quick fix. I cannot tell exactly what needs to be done to fix this properly.

Reply:
In this case I'd drop this change or replace it by https://github.com/genodelabs/genode/blob/master/tool/run/image/disk#L28-L37

Reply:
This sounds fine to me.

Reply:
I added https://github.com/genodelabs/genode/pull/2829/commits/a754be14032862c7d99df2e8bc27a3e2393ced92. I removed the hard coded disk size change and used the `-L` flag for `dm` as it is also done in the file you referenced. This seems to include the files in `var/run/<runfile>/genode` which are mostly symbolic links. It seems that these haven't been into consideration which fits with my observation that the problem appeared more often the more components the run files used.

Reply:
Great.